samtranslator/model/eventsources/push.py

```diff
@@ -668,6 +669,7 @@ class Api(PushEventSource):
     RequestModel: Optional[Dict[str, Any]]
     RequestParameters: Optional[List[Any]]
     TimeoutInMillis: Optional[PassThrough]
+    TestAuthOverride: Optional[bool]
```

Comment:
Same comment as above, Shouldn't this go under the list of [auth properties](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-apifunctionauth.html)?